package.json

```diff
@@ -22,40 +22,41 @@
   "keywords": [
     "typescript",
     "angular",
-    "angular6",
+    "angular8",
+    "ng",
     "ng6",
     "ngx",
     "embed-video",
     "embed",
-    "typescript",
```

Comment:
Why remove typescript tag?

Reply:
I removed it as it's duplicated. It's at the top of this list

Reply:
Good catch! 👌